docs/en/query_language/alter.md

```diff
@@ -283,6 +285,22 @@ For the query to run successfully, the following conditions must be met:
 - Both tables must have the same structure.
 - Both tables must have the same partition key.
 
+#### MOVE PARTITION {#alter_move-partition}
+
+``` sql
+ALTER TABLE table_source MOVE PARTITION partition_expr TO TABLE table_dest
+```
+
+This query move the data partition from the `table_source` to `table_dest` with deleting the data from `table_source`.
+
+For the query to run successfully, the following conditions must be met:
+
+- Both tables must have the same structure.
+- Both tables must have the same partition key.
+- Both tables must be the same engine family. (replicated or non-replicated)
+
```

Comment:
didn't read the code but probably that movement can work only on the same disk. I.e. the target table should have 'compatible' storage_policy.

Reply:
I tried to move a partition who have two different storage_policy, the partition was properly moved, do you have a specific edge case in mind ?

Reply:
Alright, I was able to reproduct the bug, didn't happened when I insert it, but happened when I restart the server.
I will do a fix

Reply:
Actually it's not neccessary to have exactly the same policy, but target policy should list all disks where original parts are stored.

But that way it's also safe. 

/cc @excitoon